versioned_docs/version-latest/farming-&-staking/farming/advanced-cli/cli-install.mdx

```diff
@@ -35,7 +35,7 @@ There are multiple versions of our releases available below.
 
 :::note
 Please note that a wallet with a Substrate address is required to run the software.<br />
-To learn how to create a Substrate wallet, please visit our pages on [SubWallet (recommended)](../../wallets/subwallet.md) or [Polkadot.js.](../../wallets/polkadot.md)
+To learn how to create a Substrate wallet, please visit our pages on [SubWallet (recommended)](versioned_docs/version-latest/farming-&-staking/wallets/subwallet.md) or [Polkadot.js](versioned_docs/version-latest/farming-&-staking/wallets/polkadot.md).
```

Comment:
This latest version is correct, let's just copy this to pre-release until Randy removes it